src/appengine/handlers/reproduce_tool/configure.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Allow users to configure the reproduce tool to point to this site."""
+
+import urllib
```

Comment:
please use python3 style imports here. 

you can use
```
from future import standard_library
standard_library.install_aliases()
```

To install python3 aliases.

Reply:
Done